ruby/content.md

```diff
@@ -11,7 +11,7 @@ Ruby is a dynamic, reflective, object-oriented, general-purpose, open-source pro
 ## Create a `Dockerfile` in your Ruby app project
 
 ```dockerfile
-FROM %%IMAGE%%:3.0
+FROM %%IMAGE%%:3.3
```

Comment:
Perhaps it would be better to just use the `latest` tag in these examples?

Reply:
Using an explicit version was intentional so that we're modelling the appropriate behavior for end-users who _shouldn't_ be pinning to `:latest` here -- this is a persistent issue though that we've considered changing our template engine for so that we could have a per-image calculation of this "best example tag" value.  (Barring that, IMO this change is correct :heart:)